angular-workspace/components/annotation-button/annotation-button.angular-template.ts

```diff
@@ -0,0 +1,29 @@
+import { AnnotationButton } from "dso-toolkit";
+
+import { ComponentImplementation } from "../../templates";
+
+const defaultPropValues = {
```

Comment:
Is dit nodig? Dit voelt conflicterend met de defaultValues die we in stories.ts zetten. (en default values voor optionele props worden vaak in het component al gezet)

Reply:
Dit is een patroon wat in veel andere templates ook wordt gedaan. We laten het even en kijken hier kritischer naar in #2027